tests/local/transformers/test_clean_column_names_transformer.py

```diff
@@ -0,0 +1,79 @@
+import pyspark.sql.types as T
+from spetlrtools.testing import DataframeTestCase
+
+from spetlr.spark import Spark
+from spetlr.transformers import CleanColumnNamesTransformer
+
+
+class TestCleanColumnNamesTransformer(DataframeTestCase):
+    """
+    Testing the CleanColumnNamesTransformer.
+    """
+
+    input_schema = T.StructType(
```

Comment:
I think some cases are missing in the test.
1. The keep_char "_" is not removed.
2. "-" is replaced by "_"

Only spaces and special characters "(" are tested.
Just a thought, what about numbers?

Reply:
I have create more tests